product_weight/__openerp__.py

```diff
@@ -20,24 +21,18 @@
 #
 ##############################################################################
 
-{
-    "name": "Product Weight Calculation",
-    "version": "0.1",
-    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
-    "website": "http://www.savoirfairelinux.com",
-    "license": "AGPL-3",
-    "category": "Warehouse",
-    "description": """
-    This module updates product net weight based on it's components weight
-    """,
-    "depends": [
-        "base",
-        "mrp",
-    ],
-    "demo": [],
-    "data": [
-        "wizard/product_weight_update_view.xml",
-        "product_view.xml",
-    ],
-    'installable': False,
-}
+{'name': 'Product Weight Calculation',
+ 'version': '0.2',
```

Comment:
This is a migration, Odoo already adds Odoo version before version number. So version bumping is not necessary in case of migration. But ok we can keep it to 0.2